msg/AutotuneAttitudeControlStatus.msg

```diff
@@ -17,19 +19,21 @@ float32[3] rate_sp
 float32 u_filt
 float32 y_filt
 
-uint8 STATE_IDLE = 0
+uint8 state                               # [@enum STATE]
```

Comment:
FYI `[@enum Xxx]` identifies a prefix for the values that can be used with the field. By convention the values are listed after the field. 
If you can provide useful documentation for these then great. Especially if there is anything that isn't obvious from the name.